content/docs/Reference/env-variables.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Environment Variables"
```

Comment:
We use sentence case for all OpenCue documentation, so this should be:

Environment variables